src/ui/base-page.js

```diff
@@ -11,19 +11,19 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
 
 (function ($, fluid) {
 
-    fluid.defaults("sjrk.storyTelling.page", {
+    fluid.defaults("sjrk.storyTelling.base.page", {
         gradeNames: ["fluid.modelComponent"],
         model: {
             uiLanguage: "en" //initial state is English
         },
         pageSetup: {
             resourcePrefix: "",
-            savingEnabled: true // should match the setting in sjrk.storyTelling.server.server.globalConfig
+            authoringEnabled: true // should match the setting in sjrk.storyTelling.server.config.json5
```

Comment:
The current comment is wrong. Please move this line into a commented section and explain it must be provided by integrators and what it is for.